packages/@react-spectrum/searchfield/docs/SearchField.mdx

```diff
@@ -0,0 +1,174 @@
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/searchfield';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import Refresh from '@spectrum-icons/workflow/Refresh';
+import {SearchField} from '@react-spectrum/searchfield';
+```
+
+# SearchField
+
+<p>{docs.exports.SearchField.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['SearchField']} />
+
+## Example
+
+```tsx example
+<SearchField
+  label="Search"
+  placeholder="Enter text"
+  onSubmit={text => alert(`Search for ${text}`)} />
+```
+
+## Value
+
+A SearchFields's `value` is empty by default, but an initial, uncontrolled, value can be provided using the `defaultValue` prop.
+Alternatively, a controlled value can be provided using the `value` prop.
+
+```tsx example
+<SearchField defaultValue="Default text" label="Uncontrolled SearchField" marginEnd="20px" />
+<SearchField value="Default text" label="Controlled SearchField" />
+```
+
+Placeholder text that describes the expected value or formatting for the SearchField can be provided using the `placeholder` prop. 
+Placeholder text will only appear when the SearchField is empty, and should not be used as a substitute for labeling the component with a visible label.
+
+```tsx example
+<SearchField placeholder="abc@adobe.com" label="Email" />
+```
+
+## Labeling
+
+A visual label should be provided for the SearchField using the `label` prop. If the SearchField is required, the `isRequired` and 
+`necessityIndicator` props can be used to show a required state.
+
+```tsx example
+<SearchField label="Street address" marginEnd="20px" />
+<SearchField label="Street address" isRequired necessityIndicator="icon" marginEnd="20px" />
+<SearchField label="Street address" isRequired necessityIndicator="label" marginEnd="20px" />
+<SearchField label="Street address" necessityIndicator="label" />
+```
+
+### Accessibility
+
+If a visible label isn't specified, an `aria-label` must be provided to the SearchField for 
+accessibility. If the field is labeled by a separate element, an `aria-labelledby` prop must be provided using 
+the `id` of the labeling element instead.
+
+```tsx example
+<label id="label-1">Name</label>
+<SearchField aria-labelledby="label-1" marginStart="10px" />
+```
+
+### Internationalization
+
+In order to internationalize a SearchField, a localized string should be passed to the `label` or `aria-label` prop.
+When the `necessityIndicator` prop is set to `"label"`, a localized string will be provided for `"(required)"` or `"(optional)"` automatically.
+
+```tsx example
+<SearchField label="Street address" isRequired necessityIndicator="label" marginEnd="20px" />
```

Comment:
Should these be in a provider that changes the language to show a localized string? If so, should the label should be be a localized value as well?

Reply:
yes, see the first sentence. Or am i missing your point?

Reply:
I'm wondering about changing the example to be like https://github.com/adobe-private/react-spectrum-v3/pull/197/files#diff-ae417a798db001f2a7e68cdfe7a842a9R84-R95